backend/controllers/bulkAppointmentsUpload.js

```diff
@@ -63,12 +63,9 @@ const bulkAppointmentsUploadFactory = (csvParserService, offenderLoader, logErro
             return res.redirect('/bulk-appointments/no-appointments-added?reason=offendersNotFound')
           }
 
-          if (prisonerList && prisonerList.length) {
-            req.session.data.prisonersListed = prisonerList
-          }
-          if (offenderNosNotFound && offenderNosNotFound.length) {
-            req.session.data.prisonersNotFound = offenderNosNotFound
-          }
+          req.session.data.prisonersListed = prisonerList
+
```

Comment:
Could these be undefined instead of empty lists? Is it safer to do:
```
req.session.data.prisonersListed = prisonerList || []
``` 

and same for the below?

Reply:
As discussed :)